cli/up.go

```diff
@@ -33,8 +36,11 @@ func UpRun(r *cmd.Root, c *cmd.Sub) {
 		configPath = "/etc/hyprspace/" + ifName + ".json"
 	}
 
+	log.SetLogLevelRegex("hyprspace*", InfoLogLevel)
```

Comment:
```suggestion
	log.SetLogLevel("hyprspace", "info")
	log.SetLogLevelRegex("^hyprspace/", "info")
```

This would match `hyprspaceeeeee`, but not most of the actual logger names in use. It seems like the pattern matching works by finding partial matches instead of requiring the expression to match the entire string.